.github/workflows/nightly-release.yml

```diff
@@ -269,6 +269,27 @@ jobs:
       pull-requests: write
 
     steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: 'true'
+          ref: 'rom-1.0.2'
```

Comment:
I think we want to ability to build all the fw at a specific revision (e.g. via configuration when manually running the job). But I think we still want the automated nightly release to build from head. Otherwise we won't catch any regressions in ROM until we go to build a new release.

Reply:
Okay, I think that I got a little lost in all the workflow files. Shouldn't that be handled by build-test.yml? And if this workflow file doesn't generate the release, which one does? This is the only file to call build_release.sh.